# 緊急停止モードに移行(SCIBOT-Xのみ)
---

